File: class04/cpp/AP0408-TabelaVerdade.cpp
#include <iostream>
int main(){bool v[2]={true,false};std::cout<<"A B | AeB | AouB\n";for(int i=0;i<2;i++)for(int j=0;j<2;j++){bool a=v[i],b=v[j];std::cout<<a<<" "<<b<<" | "<<(a&&b)<<"   | "<<(a||b)<<"\n";}return 0;}

File: class04/java/AP0408-TabelaVerdade.java
public class TabelaVerdade {
    public static void main(String[] x) {
        boolean[] v = {true, false};
        System.out.println("A B | AeB | AouB");
        for (boolean a : v) for (boolean b : v)
            System.out.println((a?1:0)+" "+(b?1:0)+" | "+((a&&b)?1:0)+"   | "+((a||b)?1:0));
    }
}
